plugins/node/opentelemetry-instrumentation-cassandra/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@opentelemetry/instrumentation-cassandra-driver",
+  "version": "0.20.0",
```

Comment:
should be 0.21 now

Reply:
Upgraded to 0.22 :slightly_smiling_face: